### ISSUE TITLE
Broken emisisve textures for multiblocks (immersive engineering for example)

### ISSUE THREAD

**Issue by mrvisert** [MoreMcmeta/emissive-plugin#18]
Hello! I'm making a resource pack with animated and emmisive textures for the Immersive Engineering mod. I came across the fact that when I try to set an emmisive texture to a multiblock (in particular, a coke oven), each block of the multiblock begins to have a 3D model of the ENTIRE multiblock (it’s clearer in the screenshots).

Screenshoots:
1) ![image](https://github.com/MoreMcmeta/core/assets/113045062/cbe76ec9-7e4f-45d5-893c-cecbaab564f8) 
no emmisive
2) ![image](https://github.com/MoreMcmeta/core/assets/113045062/a32e16d6-56f8-4b53-8a39-933b729443fb)
with emmisive (Just copies the multiblock model to each block in the structure)

- [ 1.19.2] Forge

ModList:
-Immersive Engineering

**Comment by soir20** [comment]
Thanks for reporting. Just FYI, I will likely not get a chance to take a look at this until the weekend

**Comment by soir20** [comment]
Since this is an emissive plugin issue, I will transfer this to that repository's issue tracker.

**Comment by soir20** [comment]
Sorry for the delayed reply; I haven't had a lot of time to look at this, and it's a tricky issue.

I'm not able to reproduce the behavior you're seeing, though I am seeing odd behavior with emissive textures on the coke oven. I can see an emissive overlay that works perfectly fine **after one texture reload, without** adding the `coke_oven.png`, `coke_oven_off.png`, and `coke_oven_on.png` base textures to my emissive pack:
![2024-03-02_17 52 23](https://github.com/MoreMcmeta/emissive-plugin/assets/71418127/4c42ca4b-e56c-43c2-a11a-f5f59756b6a2)


**After one texture reload with** base textures in the resource pack:
![2024-03-02_18 04 50](https://github.com/MoreMcmeta/emissive-plugin/assets/71418127/4633d6ed-924d-4a3a-a0be-bfaf732750c4)


I'm using MoreMcmeta v4.4.7, emissive plugin v2.0.4, and Immersive Engineering 1.19.2-9.2.3-167 in a Forge dev environment.

Can you upload the resource pack you're using so I can try to reproduce it that way? If you're not using the latest versions listed above, can you also try updating to check if you can reproduce what I'm seeing?

The emissive textures only working after a texture reload doesn't make me optimistic that this can be fixed. It suggests that Immersive Engineering is bypassing the standard texture loading process, likely in a way that makes it impossible for MoreMcmeta to work with those specific textures.

**Comment by soir20** [comment]
> The emissive textures only working after a texture reload doesn't make me optimistic that this can be fixed. It suggests that Immersive Engineering is bypassing the standard texture loading process, likely in a way that makes it impossible for MoreMcmeta to work with those specific textures.

Actually, it looks like it only works after a reload because of the way Forge caches resources, not Immersive Engineering.

**Comment by soir20** [comment]
> After one texture reload with base textures in the resource pack:

This issue on my end was caused by an improper ordering of my resource packs. I've also added a note to address the caching problem with Forge to the [wiki](https://github.com/MoreMcmeta/core/wiki/User-Docs:-Emissive-Format#overlays-for-mod-textures).

However, that doesn't solve the problem you're seeing; it just means that everything looks normal on my end.

> Can you upload the resource pack you're using so I can try to reproduce it that way? If you're not using the latest versions listed above, can you also try updating to check if you can reproduce what I'm seeing?

I'll reiterate this so that it doesn't get lost in my other messages.
